src/server/types.ts

```diff
@@ -26,6 +26,7 @@ export interface QwikDocument extends Document {}
  */
 export interface DocumentOptions {
   url?: string;
+  params?: Record<string, string>;
```

Comment:
Couldn't we get the params from the `url` here? I was assuming the `url` would be a full `URL`, to include the origin and params. So internally we could do `new URL(opts.url)`.

Reply:
good, point!  let me look into it.

Reply:
removed